### PR TITLE
Weighted aggregated statistics

### PR DESCRIPTION
**Pull request by j3r3m1**

### CODE REVIEW

**Comment by j3r3m1** [comment]
Note that the name of the indicator produced by the IProcess does not fit with the defined name in the MapuceIndicator document.

File: geoindicators/src/main/groovy/org/orbisgis/BlockIndicators.groovy [hunk 1]
```diff
@@ -0,0 +1,122 @@
+package org.orbisgis
+
+import com.sun.org.apache.xpath.internal.functions.FuncFalse
```
Comment (ebocher) [inline geoindicators/src/main/groovy/org/orbisgis/BlockIndicators.groovy]: What's this ?

File: geoindicators/src/main/groovy/org/orbisgis/BlockIndicators.groovy [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+import org.orbisgis.datamanagerapi.dataset.ITable
+import org.orbisgis.processmanagerapi.IProcess
+
+@BaseScript Geoclimate geoclimate
+
+/**
+ * This process is used to compute basic statistical operations on a specific variable from a lower scale (for
+ * example the sum of each building volume constituting a block to calculate the block volume)
+ * @return A database table name.
+ * @author Erwan Bocher
```
Comment (ebocher) [inline geoindicators/src/main/groovy/org/orbisgis/BlockIndicators.groovy]: J B

File: geoindicators/src/main/groovy/org/orbisgis/BlockIndicators.groovy [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                    if(operation=="AVG"){
+                        query += "${"weighted_avg_"+inputToTransfo},"
+                    }
+                    if(operation=="STD"){
+                        query += "${"weighted_std_"+inputToTransfo},"
+                    }
+                }
+                query += " $inputIdUp FROM $weighted_all"
+                logger.info("Executing $query")
+                println(query)
```
Comment (ebocher) [inline geoindicators/src/main/groovy/org/orbisgis/BlockIndicators.groovy]: remove println

**Comment by ebocher** [comment]
Conflict in merge

**Pull request merged**